app/api/api_v1/endpoints/lms_router.py

```diff
@@ -17,13 +20,14 @@ class GradeUpload(BaseModel):
 class UploadGradesBody(BaseModel):
     grades: List[GradeUpload]
 
-@router.post("/lms/downsync")
+@router.post("/lms/downsync", response_model=UUID)
 async def downsync(
     *,
     db: Session = Depends(get_db),
     perm: None = Depends(PermissionDependency(UserIsInstructorPermission))
 ):
-    await LmsSyncService(db).downsync()
+    task = downsync_task.delay()
+    return task.id
```

Comment:
Will this return object have a key at all, or will it just be the UUID itself? If the latter, I feel like it should return with a key called "task_id" or something and the value is the UUID.

Reply:
what?

Reply:
Like will the response body be 
```
{ 92F8f118ef929jf-f19j1f-f19mf2-1f91 }
```
or 
```
{ 
  "task_id": "92F8f118ef929jf-f19j1f-f19mf2-1f91"
}
```
I think it should be the latter.

Reply:
no the response body is a string?

Reply:
changed to return a JobSchema of id and status.